### ISSUE TITLE
Update search corpus

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/new-salem-pelican#63]
The search corpus seems incomplete; I can only successfully get records for the years 1692 and 1693.

Related, if folks are going to update content on the site, we'll need some way to help them rebuild the search corpus. It looks like `lunrcorpus.py` is mean for this, but when I run it, I get the following:

```
  File "lunrcorpus.py", line 49, in <module>
    doc_text = ''.join(BeautifulSoup(doc_html, "lxml").findAll(
  File "/Users/jeremy/.local/share/virtualenvs/new-salem-pelican-X3l9dCmC/lib/python3.6/site-packages/bs4/__init__.py", line 198, in __init__
    % ",".join(features))
bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?
```
I've run `pipenv update` to supposedly get dependencies, but maybe that dependencies list needs to include additional things?

**Comment by jeremyboggs** [comment]
Related to #61.

**Comment by shane-et-al** [comment]
LXML dependency fixed by ed6797867fed10a309255224e6fefcb7084848b2
